model/src/main/resources/reference.yml

```diff
@@ -19,6 +19,8 @@
 # exclusive, so this file is only used to show all options and to validate the configuration.
 
 ort:
+  # To overwrite if the reports (analyzer-result.yml) already exists
```

Comment:
Just use the same wording as for the configuration property in `OrtConfiguration.kt`:
"Force overwriting of any existing output files."